dev-guide/cluster-version-operator/dev/upgrades.md

```diff
@@ -58,6 +58,8 @@ Which in practice can be described in runlevels:
 0000_90_*: reserved for any post-machine updates
 ```
 
+**Note**: If the run level for a component is not defined i.e. the manifest names do not start with prefix `0000_`, it will be assigned to run level `0000_50_*` during the OCP release process. This is done to make sure if an component is agnostic about the run level it runs at, it is run as part of `0000_50_*` as it helps to update more components in parallel which helps in reducing the overall update time.
+
```

Comment:
nit: existing L46 says "runlevel" (one word) not "run level". For consistency we should either change that one line or this change.